src/libteam/build.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+## This script is to build libteam
+##
+## USAGE:
+##   ./build.sh
+
+# Obtain libteam
+rm -rf ./libteam
+git clone https://github.com/jpirko/libteam.git
+pushd ./libteam
+git checkout -f 8c1cea0
+popd
+
+git clone https://anonscm.debian.org/git/collab-maint/libteam.git tmp
+pushd ./tmp
+git checkout -f da006f2
+popd
+mv tmp/debian libteam/
+rm -rf tmp
```

Comment:
need to have -f because it is a git folder
without -f it will prompt rm: remove write-protected regular file ‘libteam/.git/objects/* etc.